apps/expo-go/ios/Podfile.lock

```diff
@@ -335,50 +335,50 @@ PODS:
     - ExpoModulesCore
   - fast_float (6.1.4)
   - FBLazyVector (0.79.0-rc.4)
-  - FirebaseAnalytics (11.4.0):
-    - FirebaseAnalytics/AdIdSupport (= 11.4.0)
-    - FirebaseCore (~> 11.0)
+  - FirebaseAnalytics (11.11.0):
```

Comment:
it _feels_ that bumping these isn't related to this PR?

Reply:
I regenerated the Podfile.lock since I added the expo-notifications module. It's possible that FirebaseAnalytics got bumped in an earlier PR and the lock file was not regenerated